site/composer.json

```diff
@@ -53,6 +53,6 @@
   "scripts": {
     "test": "phpunit",
     "lint": "phpcs --extensions=php ./app",
-    "static-analysis": "phpstan analyze app public/index.php socket/index.php"
+    "static-analysis": "php vendor/bin/phpstan analyze"
```

Comment:
Any reason why you can't just use `phpstan analyze` here?

Reply:
Actually, this is related to my lint PR as well. 
Currently the code will run into permission error even as root, to execute it without php interpreter.
For more detail look at
https://stackoverflow.com/questions/38521329/how-to-fix-permission-denied-while-running-vendor-bin-phpunit-in-a-laravel-proje